runtime/common/BaseRestRemoteClient.h

```diff
@@ -94,6 +94,8 @@ class BaseRemoteRestRuntimeClient : public cudaq::RemoteRuntimeClient {
       "func.func(combine-quantum-alloc)",
       "canonicalize",
       "cse",
+      "convert-math-to-llvm",
```

Comment:
At a minimum, this line probably isn't needed `populateMathToLLVMConversionPatterns()` is being called in `ConvertToQIR.cpp`.

I'm still investigating whether or not there are other options like using `addPipelineConvertToQIR()`, which is common code.

Reply:
Updated in 017a196c5